image/testhelpers/fake_image.go

```diff
@@ -100,11 +102,26 @@ func (f *FakeImage) TopLayer() (string, error) {
 func (f *FakeImage) AddLayer(path string) error {
 	f.assertNotAlreadySaved()
 
-	f.layersMap["sha256:"+ComputeSHA256ForFile(f.t, path)] = path
+	f.layersMap["sha256:"+shaForFile(f.t, path)] = path
 	f.layers = append(f.layers, path)
 	return nil
 }
 
+func shaForFile(t *testing.T, path string) string {
```

Comment:
Had to duplicate here for now to remove dependency on lifecycle/testhelpers package, which doesn't compile on Windows atm